dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -601,56 +528,42 @@ OP_NOT_SHAREABLE.useraction=Do not call setTypeMap, setReadOnly or setCatalog on
 # 9330 deleted
 # 9340 deleted
 
-# 9350E      OP_NOT_VALID_IN_GT
-# 
-# TRANSLATOR Do not translate: Connection, commit, rollback, setSavepoint, releaseSavepoint
+# Do not translate: Connection, commit, rollback, setSavepoint, releaseSavepoint
 # {0}        The operation being performed (such as Connection.commit)
 OP_NOT_VALID_IN_GT=DSRA9350E: Operation {0} is not allowed during a global transaction.
 OP_NOT_VALID_IN_GT.explanation=The Application Server does not allow calls to the commit, rollback, setSavepoint or releaseSavepoint methods on the Connection while a global transaction is active.
 OP_NOT_VALID_IN_GT.useraction=Do not call the commit, rollback, setSavepoint or releaseSavepoint methods on the connection when it is in a global transaction. If a non-transactional data source (for example, non-jta-data-source for Java Persistence API) is being used, then configure non-jta-data-source in persistence.xml and also configure the nonTransactionalDataSource custom property on the data source.
```

Comment:
For the phrase, "then configure non-jta-data-source in persistence.xml", I think you should add a noun after "non-jta-data-source" to indicate what's being referred to. Also, I think you should change "persistence.xml" to the "persistence.xml" file to maintain clarity.

Reply:
This old message refers to WAS configuration rather than Liberty's configuration. I'll fix that as well.